openldap-mixin/panels.libsonnet

```diff
@@ -0,0 +1,255 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+
+{
+  new(this):
+    {
+      local t = this.grafana.targets,
+      local stat = g.panel.stat,
+      local timeSeriesPanel = g.panel.timeSeries,
+      local alertListPanel = g.panel.alertList,
+
+      // Uptime Panel
+      uptime: commonlib.panels.generic.stat.base.new(
+                'Uptime',
+                targets=[t.uptime],
+                description='The total time since the LDAP server was last started.'
+              )
+              + stat.options.withGraphMode('none')
+              + stat.options.withColorMode('value')
+              + stat.options.withJustifyMode('auto')
+              + stat.options.withOrientation('auto')
+              + stat.options.reduceOptions.withCalcs(['lastNotNull'])
+              + stat.options.reduceOptions.withFields('')
+              + stat.options.reduceOptions.withValues(false)
+              + stat.options.withTextMode('value')
+
+              + stat.standardOptions.withUnit('s')
+              + stat.standardOptions.withMin(0)
+              + stat.standardOptions.color.withMode('thresholds')
+              + stat.standardOptions.thresholds.withMode('absolute')
+              + stat.standardOptions.thresholds.withSteps([
+                stat.thresholdStep.withColor('green'),
+              ]),
+
+      // Referrals Panel
+      referrals: commonlib.panels.generic.stat.base.new(
```

Comment:
For referal, you could probably get away with just using `commonlib.panels.generic.stat.info.new(`
this would greatly simplify this panel's json, and also make it adhere to their style guideline "text would be white since the color doesn't signify anything"

Reply:
updated